SwitchSpeak-iOS/SwitchSpeak-iOS/VocabCard/VocabDBWrapper.swift

```diff
@@ -210,5 +215,38 @@ class VocabCardDB {
                 self.editCard(card, inTable: table)
             }
         }
+    }*/
+
+    //Editing cards one by one as they are changed or added by the user makes the above function obsolete
+
+    private func updateChildren(children:[VocabCard], newId:Int64, inTable table:String) {
+    	for child in children {
+    		child.parentid = newId
```

Comment:
swift does not allow us to update the iterator variable (`child` in this case) within the for loop. One alternative you can consider is to loop through an `index` from `0` to `children.count-1` and then update the variable `children[index]`